spec/patch/elasticsearch_spec.rb

```diff
@@ -15,7 +15,11 @@
   end
 
   def build_client(**options)
-    patched_module::Client.new(options)
+    if Gem.loaded_specs['opensearch-ruby']
```

Comment:
This looks reasonable, but `patched_module` should already do the right thing here right?